.github/workflows/test_pipeline.yml

```diff
@@ -26,23 +26,19 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python 3.7
+      - name: Set up Python3
         uses: actions/setup-python@v1
         with:
-          python-version: 3.7
+          python-version: '3.x'
```

Comment:
what happens in this scenario? latest version?

Reply:
latest

Reply:
I don't think there's any reason to change this, to be honest. We aren't testing pyspark (yet), but it has a hard dependency on 3.7.